frontend/src/components/BotList.svelte

```diff
@@ -215,6 +319,38 @@ function ShowSelectedBotFiles() {
   {/each}
 </div>
 
+{#if filteredBots.length === 0}
+  <span>No bots available for this category.</span>
+{/if}
+
+
+{#if filteredScripts.length !== 0}
+  <p id="scripts-header">Scripts</p>
+{/if}
+
+<div class="bots">
+  <!-- svelte-ignore a11y_no_static_element_interactions -->
+  <!-- svelte-ignore a11y_click_events_have_key_events -->
+  {#each filteredScripts as script (script.id)}
+    <div class="bot" animate:flip={{ duration: flipDurationMs }} onclick={() => toggleScript(script.id)}>
```

Comment:
todo: look at if animate:flip is needed

Reply:
I copied that from the botlist

Reply:
actually, this might be nice to have when filtering